libaleth-interpreter/CMakeLists.txt

```diff
@@ -11,13 +11,23 @@ set(
     VMConfig.h
     VMOpt.cpp
 )
+
+set(
+    dependencies
+    aleth-buildinfo
+    ethash::ethash
+    evmc::evmc
+    evmc::instructions
+    Boost::boost
```

Comment:
We link all of the boost libraries?

Reply:
The `Boost::boost` represents header-only Boost libs. We need it for u256, but mostly for Windows build.

Reply:
How come header-only libs need to be linked?

Reply:
This is modern CMake magic. The CMake targets contains other information that is going to be propagated to the targets linking them, e.g. compile flags, include dir, etc.

In this case this will set `-DBOOST_ALL_NO_LIB=1` for MSVC builds. Otherwise we will have linking issues in testeth :/